emperor/support_files/js/controller.js

```diff
@@ -141,7 +192,11 @@ EmperorController.prototype.buildUI = function() {
     enableCellNavigation: true,
     forceFitColumns: true
   };
+  /**
```

Comment:
Ok.  I think I addressed it?

On Wed, Aug 19, 2015 at 12:14 PM, Yoshiki Vázquez Baeza <
notifications@github.com> wrote:

> In emperor/support_files/js/controller.js
> <https://github.com/ElDeveloper/emperor/pull/1#discussion_r37455581>:
>
> > @@ -141,7 +192,11 @@ EmperorController.prototype.buildUI = function() {
> >      enableCellNavigation: true,
> >      forceFitColumns: true
> >    };
> > +  /**
>
> Yes, this line is correct but the lines below are missing it, I should
> have made that clearer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ElDeveloper/emperor/pull/1/files#r37455581>.
>